src/index.js

```diff
@@ -36,13 +42,13 @@ class LambdaOffline {
           const invocationType = req.headers['x-amz-invocation-type'];
           const { functionName } = req.params;
 
-          const handlers = Object.keys(this.service.functions).reduce((acc, key) => {
-            const fun = this.service.getFunction(key);
-            const funOptions = functionHelper.getFunctionOptions(fun, key, servicePath, serviceRuntime);
-            const handler = functionHelper.createHandler(funOptions, {});
-            acc[key] = handler;
-            return acc;
-          }, {});
+          const options = funOptions[functionName];
+
+          if (!options) {
+            return reply().code(404);
+          }
+
+          const handler = functionHelper.createHandler(options, {});
 
           const handler = handlers[functionName];
```

Comment:
This `handler` constant is not necessary anymore, we should remove it

Reply:
Updated. Would you be open to adding a linter to help detect things like that?

Reply:
We already have an ESlint configuration and npm script to run it: https://github.com/marconi1992/serverless-offline-lambda/blob/master/package.json#L11

I'd like to add a CI/CD. I'll create an issue to address that.